[2023-10-20 23:21:10,225] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-11-15T00:00:00+00:00 [queued]>
[2023-10-20 23:21:10,241] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-11-15T00:00:00+00:00 [queued]>
[2023-10-20 23:21:10,241] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:10,243] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:21:10,243] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:10,264] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-11-15 00:00:00+00:00
[2023-10-20 23:21:10,269] {standard_task_runner.py:52} INFO - Started process 1215 to run task
[2023-10-20 23:21:10,273] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-11-15T00:00:00+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpvgbgq77q', '--error-file', '/tmp/tmp80qi0mm5']
[2023-10-20 23:21:10,275] {standard_task_runner.py:80} INFO - Job 327: Subtask segunda_tarea
[2023-10-20 23:21:10,364] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-11-15T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:21:10,463] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-11-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-15T00:00:00+00:00
[2023-10-20 23:21:10,464] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:21:10,465] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:21:10,475] {subprocess.py:85} INFO - Output:
[2023-10-20 23:21:10,477] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:21:10,478] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:21:10,513] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20221115T000000, start_date=20231020T232110, end_date=20231020T232110
[2023-10-20 23:21:10,565] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:21:10,600] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
